java/client/src/org/openqa/selenium/remote/ProtocolHandshake.java

```diff
@@ -77,13 +77,13 @@ public Result createSession(HttpClient client, Command command)
       parameters = new StringBuilder("{");
       amendW3CParameters(parameters, des, req);
       parameters.append("}");
-      LOG.info("Falling back to straight W3C remote end connection");
+      LOG.fine("Falling back to straight W3C remote end connection");
       result = createSession(client, parameters);
     }
 
     if (result.isPresent()) {
       Result toReturn = result.get();
-      LOG.info(String.format("Detected dialect: %s", toReturn.dialect));
+      LOG.fine(String.format("Detected dialect: %s", toReturn.dialect));
```

Comment:
That is much more complicated and I wouldn't merge that. Only standard Java logging should be used.